### ISSUE TITLE
Training dataset count is not correct.

### ISSUE THREAD

**Issue by shreyasvedpathak** [ultralytics/yolov5#5150]
## ❔Question
Why is training dataset count wrong but Test set is correct?

## Additional context
I am not new to using YoloV5 repo for object detection and I have used this repo for multiple projects now with different type of data, but I am facing this issue for the first time. I have images in the train and test folder, on my google drive, and the count of train images is 3000+ and test is 900+, but when I start training (refer below screenshot), the count of train data is not correct.

Images format is png, all images are 416 x 416, so I had changed the --img 480 input to --img 420, I haven't changed anything else.

![image](https://user-images.githubusercontent.com/68227226/136955316-1642d788-8855-459d-b296-e819196ae058.png)

**Comment by glenn-jocher** [comment]
@shreyasvedpathak your folder contents are your folder contents, not sure what you want me to say. YOLO will include all images with valid suffixes:
https://github.com/ultralytics/yolov5/blob/26784afd998937811dfff34392ccd83f87e1e41f/utils/datasets.py#L34-L37

Irrespective of this, you should never train on a network drive due to performance issues and connectivity risk.
